QuickOSM/ui/base_overpass_panel.py

```diff
@@ -88,6 +89,7 @@ def sort_nominatim_places(existing_places: list, place: str) -> list:
         if place in existing_places:
             existing_places.pop(existing_places.index(place))
         existing_places.insert(0, place)
+        existing_places = list(dict.fromkeys(existing_places))
```

Comment:
I though this was covered by a test, no?

Reply:
Indeed, it is just a better protection because the problem of duplicated places came from the function init nominatim autofill